### DISCUSSION TITLE
Is it possible to run several simulations in parallel?

### DISCUSSION THREAD

**Discussion by DeeDive** [aidudezzz/deepbots#113] [Q&A] [answered]
Dear authors!

Thanks for the great interface. I am wondering that is it possible to run several simulations in parallel? Or maybe this is more of a question related to webots. By doing that we can speed up the evaluations significantly.

Millions of thanks in advance!

**Comment by tsampazk** [comment, marked as answer]
Hey @DeeDive! Thank you for your interest in deepbots. Unfortunately, no, deepbots doesn't support parallel simulations by default. 

You could use deepbots to build something though, e.g. using the [emitter-receiver scheme](https://github.com/aidudezzz/deepbots#emitter---receiver-scheme) and add multiple robots to a world and gather experience from all of them simultaneously, speeding up the training. This would require creating a communication scheme between the robots and the supervisor.

You can take a look at the work done on [this PR](https://github.com/aidudezzz/deepworlds/pull/48).
